Examples/Algorithms/Vertexing/include/ActsExamples/Vertexing/AdaptiveMultiVertexFinderAlgorithm.hpp

```diff
@@ -56,14 +57,13 @@ class AdaptiveMultiVertexFinderAlgorithm final : public IAlgorithm {
   using Linearizer = Acts::HelicalTrackLinearizer<Propagator>;
   using Fitter =
       Acts::AdaptiveMultiVertexFitter<Acts::BoundTrackParameters, Linearizer>;
-  using Seeder = Acts::TrackDensityVertexFinder<
-      Fitter, Acts::GaussianTrackDensity<Acts::BoundTrackParameters>>;
-  using Finder = Acts::AdaptiveMultiVertexFinder<Fitter, Seeder>;
   using Options = Acts::VertexingOptions<Acts::BoundTrackParameters>;
 
   using VertexCollection =
       std::vector<Acts::Vertex<Acts::BoundTrackParameters>>;
 
+  enum SeedFinder { GaussianSeeder, AdaptiveGridSeeder };
```

Comment:
```suggestion
  enum class SeedFinder { GaussianSeeder, AdaptiveGridSeeder };
```

should be preferred as it provides better type safety and does not pollute the namespace